daemons/controld/controld_join_dc.c

```diff
@@ -774,23 +786,25 @@ do_dc_join_ack(long long action,
     }
 
     peer = pcmk__get_node(0, join_from, NULL, pcmk__node_search_cluster_member);
-    if (peer->join != crm_join_finalized) {
+    if (controld_get_join_phase(peer) != controld_join_finalized) {
+        enum controld_join_phase phase = controld_get_join_phase(peer);
```

Comment:
might as well do this before the if

Reply:
Yeah, I just figured I'd save myself a variable in the broader scope, since this is a short inline function.